docs/source/installation/index.rst

```diff
@@ -33,33 +33,82 @@ Download the source
+Manual installation with editable mode requires you to install the packages in order to avoid potential conflicts or missing dependencies.
+For the correct order refer to the tables *requires* column.
+
+Installing them without edible mode wont require the right order since it will automatically install dependencies.
+Each package can be installed using: ::
+
+    pip install <package_name>
+
+**For Students of the CI group:**
+Packages with a **!** next to them are most likely required.
+You can also use a shortcut to install all required packages: ::
+
+    sh student_install.sh
+
+This script installs all required packages in editable mode.
+
+
+
+.. list-table:: revolve2 packages
+   :widths: 25 50 25 5
+   :header-rows: 1
+
+   * - Package Name
+     - Functionality
+     - requires
+     -
+   * - actor_controller
+     - This package provides a controller interface as well as already implemented controllers for the modular robots.
+     - :code:`serialization`
+     - **!**
+   * - ci_group
+     - This package provides revolve2 with CI group specific revolve configuration and helper tools.
+     - :code:`simulation` & :code:`modular_robot`
+     - **!**
+   * - experimentation
+     - This package provides functionality for experiments, such as optimization techniques, genotype operations and database usage.
+     - :code:`modular_robot`
+     - **!**
+   * - modular_robot
+     - This package provides revolve2 with all functionality around the robots and their modules.
+     - :code:`simulation` & :code:`actor_controller`
+     - **!**
+   * - rpi_controller
+     - This package allows to run a revolve2 ``ActorController`` on a physical robot, with the same behavior as in the simulations.
+     - :code:`actor_controller`
+     -
+   * - rpi_controller_remote
+     - This package allows to remotely connect to, and run a physical robot using SSH.
+     - :code:`serialization`
+     -
+   * - serialization (Deprecated)
+     - This package does what the name says. (Will be removed soon)
+     -
+     - **!**
+   * - simulation
+     - This package provides an abstraction layer for physics simulators. Other packages provide a simulator-specific implementation, such as for MuJoCo.
+     - :code:`actor_controller`
+     - **!**
+   * - simulators/mujoco
+     - This package provides simulation using the MuJoCo simulator.
+     - :code:`simulation` & :code:`modular_robot`
+     - **!**
 
-.. toctree::
-   :maxdepth: 1
 
-   Mujoco physics runner <runners/mujoco>
-   CPPNWIN genotype <genotypes/cppnwin>
-   Raspberry Pi actor controller <rpi_controller>
 
 -------------
 Editable Mode
 -------------
-If you want to edit revolve2's code while having it installed, consider using pip's ``editable mode``::
+When developing element in python packages it is crucial to test changes iteratively. To avoid constant :code:`pip uninstall` and :code:`pip install` spam, you can simply use pip`s built in "developer" mode.
```

Comment:
```suggestion
When developing element in python packages it is crucial to test changes iteratively. To avoid having to constantly :code:`pip uninstall` and :code:`pip install`, you can simply use pip`s built in "developer" mode.
```